### PR TITLE
[cli] add cli command to control child supervision

### PR DESCRIPTION
**Pull request by Irving-cl**
This PR adds cli command to control child supervision in order to replacing `wpanctl` with `ot-ctl` in silk.

### CODE REVIEW

File: src/cli/cli.cpp [hunk 1]
```diff
@@ -1032,6 +1035,58 @@ otError Interpreter::ProcessChildMax(uint8_t aArgsLength, char *aArgs[])
...
+
+    if (strcmp(aArgs[0], "checktimeout") == 0)
+    {
+        if (aArgsLength == 1)
+        {
+            OutputLine("%u", otChildSupervisionGetCheckTimeout(mInstance));
+        }
+        else if (aArgsLength == 2)
+        {
+            SuccessOrExit(error = ParseLong(aArgs[1], value));
```
Comment (abtink) [inline src/cli/cli.cpp:1054]: Use newly added `ParseAsUint16`?

File: src/cli/cli.cpp [hunk 2]
```diff
@@ -1032,6 +1035,58 @@ otError Interpreter::ProcessChildMax(uint8_t aArgsLength, char *aArgs[])
...
+#if OPENTHREAD_FTD
+    else if (strcmp(aArgs[0], "interval") == 0)
+    {
+        if (aArgsLength == 1)
+        {
+            OutputLine("%u", otChildSupervisionGetInterval(mInstance));
+        }
+        else if (aArgsLength == 2)
+        {
+            SuccessOrExit(error = ParseLong(aArgs[1], value));
```
Comment (abtink) [inline src/cli/cli.cpp:1071]: Use `ParseAsUint16()?`

**Review by abtink — APPROVED** [review]
LGTM.

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**